### DISCUSSION TITLE
Enhancement: UI feature, one-click access to selected "Chat Bot"

### DISCUSSION THREAD

**Discussion by cjccjj** [danny-avila/LibreChat#4481] [Feature Requests & Suggestions]
### What features would you like to see added?

Provide consistent and uniform, one-click access for end users to select or switch to a "Chat Bot" that they can start chatting with immediatly, no matter if it is a model under an endpoint, a preconfigured assistant, a plugin setting, an agent, or a model preset. 

This list/button can be placed in the header of the main chat view. The list/button only shows "Chat Bot" pre selected by admins (and the user), and usually should be a short list. With this new UI, the side panel and current header menus can be turned off for normal users and will be accessible only to admins. (like a much simplified poe.com)

### More details

**Issue**:  
1. Currently, it is inconvenienct and inconsistent for users to switch to a "Chat Bot". It requires four or more clicks, with some needing to click on the header and others on the side panel.  
2. There is no concept in the codebase to define an object that users perceive as a "Chat Bot". This can include models, models with presets, preconfigured assistants, or agents. Regular users are interested in their names and capabilities rather than their origins.

**Suggestions**:  
1. Provide uniform access to switch to a "ready-to-chat" setting of a model, or "Chat Bot".  
2. Add labels and avatars to all "Chat Bots" (assistants and agents already have them).  
3. Implement a bookmark feature to manage which "Chat Bots" are selected to show up (or just show all that is enabled through current .env and .yaml config).  
4. Use "Chat Bot" avatars as buttons in the header for fewer than four options, or a drop-down list for more. Enable switching with one or two clicks, preferably hiding the side panel and header for regular users especially on mobile devices.

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This feature is implemented already: https://www.librechat.ai/docs/configuration/librechat_yaml/object_structure/model_specs#overview

**Reply by cjccjj** [reply]
I'm sorry for missing that feature. I have no idea how I overlooked it. I appreciate your quick response!
